packages/frontend/src/api/utils/portfolioUtils.test.ts

```diff
@@ -0,0 +1,53 @@
+import { TPortfolio } from "../types";
+import { getAssetPrefix } from "./portfolioUtils";
+
+describe("getAssetPrefix", () => {
+    it("should return empty string for ethereum network", () => {
+        const portfolio = {
+            network: "ethereum",
+
+            // Add other required TPortfolio properties
+        } as TPortfolio;
```

Comment:
Dont you think doing type assertions like this could give false positives?

Reply:
Please explain a scenario